metadata.json

```diff
@@ -90,11 +90,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.1.0 < 4.0.0"
     },
     {
       "name": "puppet",
-      "version_requirement": "2.7.0"
+      "version_requirement": ">= 3.1.0 < 4.0.0"
```

Comment:
Since we are not testing against Puppet v4, we don't know that. Suggest merging this and then changing to include v4 along with adding it to the test matrix.